sssom/util.py

```diff
@@ -812,14 +851,25 @@ def curie_from_uri(uri: str, prefix_map: Mapping[str, str]) -> str:
 
 
 def get_prefixes_used_in_table(df: pd.DataFrame) -> List[str]:
+    """Get prefixes used in table.
+
+    :param df: Pandas DatafFrame
+    :return: List of unique prefixes
```

Comment:
this function is simple enough that the param and return values are redundant. just use a single line